web/templates/settings/admin_linkifier_list.hbs

```diff
@@ -14,10 +14,10 @@
     </td>
     {{#if ../can_modify}}
     <td class="no-select actions">
-        <button class="button small edit btn-warning" data-linkifier-id="{{id}}" title="{{t 'Edit' }}" aria-label="{{t 'Edit' }}">
+        <button class="button small edit btn-warning tippy-zulip-delayed-tooltip" data-linkifier-id="{{id}}" data-tippy-content="{{t 'Edit' }}" aria-label="{{t 'Edit' }}">
             <i class="fa fa-pencil"></i>
         </button>
-        <button class="button small delete btn-danger" data-linkifier-id="{{id}}" title="{{t 'Delete' }}" aria-label="{{t 'Delete' }}">
+        <button class="button small delete btn-danger tippy-zulip-delayed-tooltip" data-linkifier-id="{{id}}" aria-label="{{t 'Delete' }}" data-tippy-content="{{t 'Delete' }}" >
```

Comment:
I think the final space at the end (` >` ) isn't necessary or consistent with our usual style?